scaleway/helpers_redis.go

```diff
@@ -106,3 +135,55 @@ func flattenRedisSettings(settings []*redis.ClusterSetting) interface{} {
 	}
 	return rawSettings
 }
+
+func flattenRedisPrivateNetwork(endpoints []*redis.Endpoint) (interface{}, bool) {
+	pnFlat := []map[string]interface{}(nil)
+	for _, endpoint := range endpoints {
+		if endpoint.PrivateNetwork != nil {
```

Comment:
Could you transform this like this:

```go
if endpoint.PrivateNetwork == nil {
    continue
}
...
```

by doing so, we are able to remove a level of indentation.